services/provider/server/server.go

```diff
@@ -799,12 +794,25 @@ func (s *OCSProviderServer) ReportStatus(ctx context.Context, req *pb.ReportStat
 		return nil, status.Errorf(codes.Internal, "Failed to update lastHeartbeat payload in the storageConsumer resource: %v", err)
 	}
 
+	storageConsumer, err := s.consumerManager.Get(ctx, req.StorageConsumerUUID)
+	if err != nil {
+		return nil, status.Errorf(codes.Internal, "Failed to get storageConsumer resource: %v", err)
+	}
+
 	channelName, err := s.getOCSSubscriptionChannel(ctx)
 	if err != nil {
 		return nil, status.Errorf(codes.Internal, "Failed to construct status response: %v", err)
 	}
+	var arr = make([]string, 0)
+	arr = append(arr, "DesiredClientOperatorChannel", channelName)
+	if storageConsumer.Spec.StorageQuotaInGiB > 0 {
+		clusterResourceQuotaSpec := getClusterResourceQuotaSpec(storageConsumer)
+		arr = append(arr, "ClusterResourceQuotaSpec", fmt.Sprintf("%v", clusterResourceQuotaSpec))
+	}
```

Comment:
As of the last 3 comments, please change the code to:
```suggestion

	var arr = []any {
		channelName,
		storageConsumer.Spec.StorageQuotaInGiB,
	}

```